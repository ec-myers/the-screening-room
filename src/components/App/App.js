import React, { Component } from 'react';
import Nav from '../Nav/Nav';
import LoginForm from '../LoginForm/LoginForm';
import Container from '../Container/Container';
import SelectedMovie from '../SelectedMovie/SelectedMovie';
import { getMovies } from '../../apiCalls/apiCalls';
import './App.scss';

class App extends Component {
  constructor() {
    super();
  }

  componentDidMount = async () => {
     getMovies()
  }

  render() {
    return (
      <>
        <h1>Screening Room</h1>
        <Nav />
        <LoginForm />
        <Container />
        <SelectedMovie />
      </>
    );
  }
}

export default App;
