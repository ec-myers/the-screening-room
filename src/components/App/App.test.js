import React from 'react';
import { shallow } from 'enzyme';
import { App, mapStateToProps, mapDispatchToProps } from './App';

describe('App', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<App />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
