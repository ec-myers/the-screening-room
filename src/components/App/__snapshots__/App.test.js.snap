// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot 1`] = `
<main
  className="main"
>
  <Connect(Nav) />
  <Route
    path="/movies/:id"
    render={[Function]}
  />
  <Route
    path="/(|movies|signup|login)"
    render={[Function]}
  />
  <Route
    path="/(login|signup)"
    render={[Function]}
  />
  <Route
    exact={true}
    path="/favorites"
    render={[Function]}
  />
  <Footer />
</main>
`;
